app/models/publication.rb

```diff
@@ -230,15 +230,19 @@ def year
   end
 
   def sciencewire_pub?
-    provenance == 'sciencewire'
+    provenance == Settings.sciencewire_source
   end
 
   def pubmed_pub?
-    provenance == 'pubmed'
+    provenance == Settings.pubmed_source
   end
 
   def wos_pub?
-    provenance == 'wos'
+    provenance == Settings.wos_source
```

Comment:
we have these strings in the settings already, let's use them instead of duplicating